ci/release/update-version.sh

```diff
@@ -36,6 +36,8 @@ function sed_runner() {
 # RTD update
 sed_runner 's/version = .*/version = '"'${NEXT_SHORT_TAG}'"'/g' docs/source/conf.py
 sed_runner 's/release = .*/release = '"'${NEXT_FULL_TAG}'"'/g' docs/source/conf.py
+# docs update
+sed_runner "s/cuxfilter=${CURRENT_SHORT_TAG}/cuxfilter=${NEXT_SHORT_TAG}/g" docs/source/user_guide/installation.rst
```

Comment:
```suggestion
sed_runner "/cuxfilter=[0-9]\{2\}.[0-9]\{2\}/ s/=[0-9]\{2\}.[0-9]\{2\}/=${NEXT_SHORT_TAG}/g" docs/source/user_guide/installation.rst
```

For the reasons outlined in the comment below, I discourage the use of variables in the search pattern of the `sed` expression.

- https://github.com/rapidsai/ucxx/pull/36#discussion_r1178429239